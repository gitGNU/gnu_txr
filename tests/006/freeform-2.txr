@(freeform 5 " ")
@(coll)@{a /./}@(until)1 2 @(end)
@b
