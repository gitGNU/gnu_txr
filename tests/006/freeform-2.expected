a[0]="a"
a[1]=" "
a[2]="b"
a[3]=" "
a[4]="c"
a[5]=" "
b="1"
